Tox fails if there is a virtualenv inside root dir

### Steps to reproduce this issue

1. Clone de repository
2. `cd django-config-models`
3. `virtualenv env`
4. `source env/bin/activate`
5. `pip install tox`
6. `tox py27-django{18,19}`

### Possible fix

Explicitly tell `py.test` to run on `config_models` package in the `tox.ini` file.

### Relevant error message

```
===================================================================================== FAILURES ======================================================================================
____________________________________________________________________________________ test_pydist ____________________________________________________________________________________

    def test_pydist():
        """Make sure pydist.json exists and validates against our schema."""
        # XXX this test may need manual cleanup of older wheels
    
>       import jsonschema
E       ImportError: No module named jsonschema

env/lib/python2.7/site-packages/wheel/test/test_basic.py:117: ImportError
____________________________________________________________________________________ test_keygen ____________________________________________________________________________________

    def test_keygen():
        def get_keyring():
            WheelKeys, keyring = tool.get_keyring()
    
            class WheelKeysTest(WheelKeys):
                def save(self):
                    pass
    
            class keyringTest:
                @classmethod
                def get_keyring(cls):
                    class keyringTest2:
                        pw = None
                        def set_password(self, a, b, c):
                            self.pw = c
                        def get_password(self, a, b):
                            return self.pw
    
                    return keyringTest2()
    
            return WheelKeysTest, keyringTest
    
>       tool.keygen(get_keyring=get_keyring)

env/lib/python2.7/site-packages/wheel/test/test_tool.py:25: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
env/lib/python2.7/site-packages/wheel/tool/__init__.py:39: in keygen
    WheelKeys, keyring = get_keyring()
env/lib/python2.7/site-packages/wheel/test/test_tool.py:5: in get_keyring
    WheelKeys, keyring = tool.get_keyring()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    def get_keyring():
        try:
            from ..signatures import keys
            import keyring
            assert keyring.get_keyring().priority
        except (ImportError, AssertionError):
>           raise WheelError("Install wheel[signatures] (requires keyring, keyrings.alt, pyxdg) for signatures.")
E           WheelError: Install wheel[signatures] (requires keyring, keyrings.alt, pyxdg) for signatures.

env/lib/python2.7/site-packages/wheel/tool/__init__.py:34: WheelError
================================================================================= warnings summary ==================================================================================
env/lib/python2.7/site-packages/wheel/test/test_signatures.py::test_ed25519py
  /Users/lucas/opencraft/django-config-models/env/lib/python2.7/site-packages/wheel/signatures/ed25519py.py:24: RuntimeWarning: ed25519ll should choose random seed.
    RuntimeWarning)

-- Docs: http://doc.pytest.org/en/latest/warnings.html
================================================================== 2 failed, 43 passed, 1 warnings in 8.52 seconds ==================================================================
ERROR: InvocationError: '/Users/lucas/opencraft/django-config-models/.tox/py27-django18/bin/py.test'
```

I'll send a PR later fixing this problem. I'm just opening this issue to keep track of it.

--- a/tox.ini
+++ b/tox.ini
@@ -20,7 +20,7 @@ deps =
     django19: Django>=1.9,<1.10
     -r{toxinidir}/requirements/test.txt
 commands =
-    py.test {posargs}
+    py.test config_models {posargs}
 
 [testenv:docs]
 setenv =

